lib/foreman/renderer/scope/macros/base.rb

```diff
@@ -334,6 +334,16 @@ def gem_version_compare(first, second)
             Gem::Version.new(first.to_s) <=> Gem::Version.new(second.to_s)
           end
 
+          apipie :method, 'Returns filtered hosts by given search query' do
+            required :hosts, 'ActiveRecord::Relation', desc: "The array of hosts"
+            required :search_query, 'String', desc: "Query that is used for filter"
+            returns Array, desc: 'list of hosts'
+            example 'search_in_hosts_relation(hosts, "operating_system = CentOS") => only hosts with CentOS'
+          end
+          def search_in_hosts_relation(hosts, search_query)
+            hosts.search_for(search_query)
+          end
```

Comment:
Is this still needed or is it a leftover from previous version? because the commit message doesn't mention this method